packages/wasm-dpp/src/errors/consensus/consensus_error.rs

```diff
@@ -376,6 +378,9 @@ fn from_basic_error(basic_error: &BasicError) -> JsValue {
         InvalidAssetLockProofTransactionHeightError(e) => {
             InvalidAssetLockProofTransactionHeightErrorWasm::from(e).into()
         }
+        InvalidIdentityCreditTransferAmountError(e) => {
+            InvalidIdentityCreditTransferAmountErrorWasm::from(e).into()
+        }
```

Comment:
Shouldn't we just add all other variants instead of  _ => todo!() ?

Reply:
I just wondering if we should do it as a separate PR so we can get this fix now?